full_install.sh

```diff
@@ -10,3 +10,4 @@ python3 -m pip install -e vision/
 python3 -m pip install -e timeseries/[all,tests]
 python3 -m pip install -e eda/
 python3 -m pip install -e autogluon/
+python3 -m pip install -e fair/
```

Comment:
Move to the line above autogluon install, autogluon/ install should always be last